app/views/debit_direct_files/show.html.haml

```diff
@@ -0,0 +1,7 @@
+.contextual
+  = icon_link_to :print, {:format => :text}, {:target => '_blank'}
```

Comment:
At least in Chrome and Firefox the target is not needed, as we send the file with a filename.